src/atomate2/common/jobs/anharmonicity.py

```diff
@@ -0,0 +1,611 @@
+"""Jobs for running anharmonicity quantification."""
+
+from __future__ import annotations
+
+import contextlib
+import logging
+from typing import TYPE_CHECKING
+
+import numpy as np
+from jobflow import Flow, Response, job
+from phonopy import Phonopy
+from pymatgen.core import Structure
+from pymatgen.core.units import kb
+from pymatgen.io.phonopy import get_phonopy_structure, get_pmg_structure
+
+from atomate2.aims.schemas.calculation import Calculation
+from atomate2.common.schemas.anharmonicity import AnharmonicityDoc
+
+if TYPE_CHECKING:
+    from pathlib import Path
+    from typing import Any
+
+    from atomate2.aims.jobs.base import BaseAimsMaker
+    from atomate2.common.schemas.phonons import ForceConstants, PhononBSDOSDoc
+    from atomate2.forcefields.jobs import ForceFieldStaticMaker
+    from atomate2.vasp.jobs.base import BaseVaspMaker
+
+
+logger = logging.getLogger(__name__)
+
+# Constants
+EV = 1.60217733e-19  # [J]
+AA = 1e-10  # [m]
+AMU = 1.66053904e-27  # [kg]
+THZ = 1e12  # [1/s]
+PI = np.pi
+OMEGA_TO_THZ = (EV / AA**2 / AMU) ** 0.5 / THZ / 2 / PI  # 15.633302 THz
+
+
+@job
+def get_phonon_supercell(phonon_doc: PhononBSDOSDoc) -> Structure:
+    """Get the phonon supercell of a structure.
+
+    Parameters
+    ----------
+    phonon_doc: PhononBSDOSDoc
+        The output of a phonopy workflow
+
+    Returns
+    -------
+    Structure
+        The phonopy structure
+    """
+    cell = get_phonopy_structure(phonon_doc.structure)
+    phonon = Phonopy(
+        cell,
+        phonon_doc.supercell_matrix,
+    )
+    return get_pmg_structure(phonon.supercell)
```

Comment:
could be implemented without the doc and only supercell matrix and structure but maybe not necessary to change it.